### ISSUE TITLE
It seems "Girls / Girls / Boys" by Panic! At The Disco breaks the app

### ISSUE THREAD

**Issue by Pixel-Rogue** [SimonIT/spotifylyrics#55]
The app doesn't seem to have any trouble loading any lyrics so far, but every time I play this song it gets stuck loading the lyrics, and then wont load lyrics for any other songs until I restart the app.

**Comment by SimonIT** [comment]
You're right, there are some lyrics provider causing some trouble. I'll try to fix them an release a new version
